lib/index.ts

```diff
@@ -1,6 +1,7 @@
 export { isNotNull } from './isNotNull';
 export { isNotUndefined } from './isNotUndefined';
 export { isNotNullish } from './isNotNullish';
+export type { NotNullish } from './isNotNullish';
```

Comment:
[suggestion] With latest TS versions, you can also do this:

```suggestion
export { type NotNullish, isNotNullish } from './isNotNullish';
```

Reply:
I didn't know you could do that, updating.